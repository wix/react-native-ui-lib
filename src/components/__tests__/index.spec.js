describe('components/index', () => {
  it('should export the same structure', () => {
    expect(require('../index')).toMatchSnapshot();
  });
});
