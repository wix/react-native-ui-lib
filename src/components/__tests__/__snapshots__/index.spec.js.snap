// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/index should export the same structure 1`] = `
Object {
  "ActionBar": [Function],
  "ActionSheet": [Function],
  "Avatar": [Function],
  "Badge": [Function],
  "Button": [Function],
  "Card": [Function],
  "Carousel": [Function],
  "Checkbox": [Function],
  "ConnectionStatusBar": [Function],
  "Dialog": [Function],
  "FeatureHighlight": [Function],
  "Image": [Function],
  "ListItem": [Function],
  "MaskedInput": [Function],
  "Notification": [Function],
  "PageControl": [Function],
  "Picker": [Function],
  "RadioButton": [Function],
  "RadioGroup": [Function],
  "Stepper": [Function],
  "Switch": [Function],
  "TabBar": [Function],
  "TagsInput": [Function],
  "Text": [Function],
  "TextArea": [Function],
  "TextField": [Function],
  "TextInput": [Function],
  "Toast": [Function],
  "TouchableOpacity": [Function],
  "Tour": [Function],
  "View": [Function],
  "WheelPickerDialog": [Function],
}
`;
